uber/templates/registration/history.html

```diff
@@ -67,3 +67,25 @@
 {% endfor %}
 
 {% endblock %}
+{% block scripts %}
+    <script>
+        var resend = function(email_id){
+            $.ajax(
+                    {
+                        method: 'POST',
+                        url:'resend_email',
+                        data:{
+                            csrf_token: csrf_token,
+                            id: email_id
+                        },
+                        success: function(response){
```

Comment:
I am unsure about displaying the message. These are UI things I am planning to solve with @cybersenshi soon™